src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <!-- set PrivateAssets=None to ensure that all assets, including Build and Analyzer, are included in the nuspec -->
+    <PackageReference Include="Microsoft.AspNetCore" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
```

Comment:
As I mentioned previously I only included the files marked as ship,include in the packages.csv in Coherence: https://github.com/aspnet/Coherence/blob/dev/packages/packages.csv#L3. Microsoft.AspNetCore was marked as exclude so I did not include it in the M.A.All package. Given that the M.A.All package is a superset of the M.A metapackage I'm not sure if this is required.